EduID/Utilities/Constants.swift

```diff
@@ -4,7 +4,7 @@ enum Constants {
     
     enum RegEx {
         static let emailRegex = #"^(?=.{6,})[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$"#
-        static let nameRegex = #"^[a-zA-Z]+(?:[a-zA-Z- ']+)*$"#
+        static let nameRegex = #"^(?!\s*$).+"# // Checks for "is-not-empty"
```

Comment:
I think this regex won't accept a name like William-Alexander?

Reply:
It will, it only rejects whitespace. If there's one non-whitespace character in the field, it will accept.